File: playground/java/HelloGoodbye.java
public class HelloGoodbye
{
    public static void main(String[] args)
    {
        if (args.length == 2)  // Check how many arguments are passed to the program.
        {
            System.out.println("Hello " + args[0] + " and " + args[1] + ".");
            System.out.println("Goodbye " + args[1] + " and " + args[0] + ".");
        }
    }
}

File: playground/csharp/HelloGoodbye.cs
using System;

namespace HelloGoodbye
{
    class Program
    {
        static void Main(string[] args)
        {
            if (args.Length == 2)  // Check how many arguments are passed to the program.
            {
                Console.WriteLine("Hello {0} and {1}.", args[0], args[1]);
                Console.WriteLine("Goodbye {1} and {0}.", args[0], args[1]);
            }
        }
    }
}
